tests/gem/test_api.py

```diff
@@ -134,18 +136,16 @@ class TestContextManager(IsolatedAsyncioTestCase):
     def setUp(self):
         self._queue: asyncio.Queue[PacketProtocolMessage] = asyncio.Queue()
         self._transport = MockTransport()
-        self._protocol = BidirectionalProtocol(self._queue)
+        self._protocol = BidirectionalProtocol(
+            self._queue, packet_delay_clear_time=timedelta(seconds=0)
+        )
         self._protocol.connection_made(self._transport)
 
     @pytest.mark.asyncio
     @patch(
         "siobrultech_protocols.gem.protocol.API_RESPONSE_WAIT_TIME",
         timedelta(seconds=0),
     )
-    @patch(
-        "siobrultech_protocols.gem.protocol.PACKET_DELAY_CLEAR_TIME",
```

Comment:
It probably should have been a flag that we might want this configurable when we had to patch this in tests.